src/database/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Client } from "pg";
+import { drizzle } from "drizzle-orm/node-postgres";
+import * as schema from "./schema";
+
+const client = new Client({
+  connectionString: process.env["DB_URL"],
```

Comment:
use the `src/lib/env` instead of process.env

Uncomment line 7 of that file and then you can use the code below, which guarantees that the variable exists *and* is a valid URL

```ts
import { env } from "@lib/env";

env.DATABASE_URL
```